### PR TITLE
Resource Page Frontend

### LINKED ISSUE #99: [Resources] [FE] Page implementation
## Context

Angels Among Us is requesting a page with useful contact information and important phone numbers so that volunteers can find a way to contact them in the event they have any issues during the foster process.

## Requirements

Build this new page at `/resources`, and update the navbar's button to navigate to the new page (currently, the "Resources" button on the navbar is not functional). Reference the Figma design **(desktop and mobile)** to build out all aspects of this page. The page should be accessible regardless of user role. The navbar should be visible.

#### PM

- [ ] Product-specific requirements
- [ ] I can use the resources button on the nav bar at any time to go to the resources page
- [ ] Emergency numbers are at the top
- [ ] The page/text/spacing is even throughout using an auto-lay out/dynamic resizing
- [ ] The page is usable and the experience is the same for desktop and mobile view

#### EM

- [ ] Uses the pageAccess HOC
- [ ] Navbar "Resources" button works properly

#### Designer

- [ ] Design-specific requirements

### PR DESCRIPTION
**Pull request by its-edmund**
## PR Title

Issue Number(s): #99

What does this PR change and why?

### Checklist

- [x] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [x] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: web/components/Navbar.tsx
Comment (katsukixyz) [inline web/components/Navbar.tsx:91]: Link the resources page on the mobile navbar as well

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Use `white` for readability

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Add this to our Chakra tokens and use the token name instead

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx:361]: Apply `marginTop` here instead of `marginBottom` on the primary contact text for spacing consistency between the team name and the children

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Use white

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Same thing as before

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Abstract into its own component

File: web/pages/resources.tsx
Comment (katsukixyz) [inline web/pages/resources.tsx]: Use `PropsWithChildren<T>` instead of manually typing children - modify `TeamProps` to just include `team`

**Review by katsukixyz — COMMENTED** [review]
Great work on this page - just a few comments relating to readability.

**Comment by rohan-chadha** [comment]
Amazing work! Just a few small things:

The image below shows that the header text (Kim Kay) and the blue div don't have enough space between them... fix spacing there...
image

When the window width is decreased, the blue divs here break, and the left most one isn't the same size as the other blue divs:
image

Last thing... it seems the font size for the descriptions under each email doesn't match the Figma font size. Try to fix this, and then we can see about decreasing other font sizes to make the resources page easier to navigate. It seems when I zoom out to 90% on Chrome, the page is easier to read, but this could just be me! Let's discuss more about this at the next general meeting!

**Comment by winniezhangg** [comment]
Awesome work! Rohan and Katsuki have covered the majority of the bases. Definitely keep in mind the specific spacing for the divs as Rohan says. Figma has a new super helpful dev mode that simplifies things.  

One more thing... I tried to optimize the layout of the different departments as closely as possible to the order of the list Angels provides. Order is perfect for mobile. If possible, reorder some to match the design to limit the awkward white space at the bottom as much as possible.

Screen Shot 2023-09-09 at 7 14 58 PM

**Comment by winniezhangg** [comment]
Mobile looks great, nothing new except same concerns as desktop!

**Comment by Wesley257** [comment]
Nit: Think there is a small typo in the description with "kimkay@angelsrescue.org". Looks like an extra space and apostrophe are there.

File: web/pages/resources.tsx
Comment (its-edmund) [reply web/pages/resources.tsx]: Wait that's crazy

**Review by katsukixyz — APPROVED** [review]
Great work! Hold off on merging this until the rest of the LT has approved it as well

**Comment by rohan-chadha** [comment]
Everything looks great!

**Pull request merged**
